.github/workflows/go.yml

```diff
@@ -59,3 +59,9 @@ jobs:
         lcov-base: ./main.lcov
         delete-old-comments: true
         github-token: ${{ secrets.GITHUB_TOKEN }}
+    - name: Static check
+      if: matrix.version == '1.18.1' && github.event_name == 'pull_request'
```

Comment:
this should run without this `if` i think.

Reply:
I think so, we should be static checking all versions imo. It may not make a difference now, but when we are targeting different major versions it may do so.

Reply:
I hope I understood: I will leave the condition for the pull_requests only then